### ISSUE TITLE
Unable to build: apk 404

### ISSUE THREAD

**Issue by hasezoey** [golemfactory/ya-runtime-vm#82]
Steps:
- `git clone thisrepo --depth 10`
- `cd ya-runtime-vm`
- `cargo build --release --all`
- Error

Error:
```ts
error: failed to run custom build command for `ya-runtime-vm v0.2.5 (/home/hasezoey/Downloads/ya-runtime-vm/runtime)`

Caused by:
  process didn't exit successfully: `/home/hasezoey/Downloads/ya-runtime-vm/target/release/build/ya-runtime-vm-fa369a1c85baae37/build-script-build` (exit code: 1)
  --- stdout
  wget -q -O "unverified" "https://nl.alpinelinux.org/alpine/v3.13/main/x86_64/linux-virt-5.10.16-r0.apk"

  --- stderr
  make: *** [Makefile:31: unpacked_kernel] Error 8
  Error: make failed with code 2
warning: build failed, waiting for other jobs to finish...
error: build failed
```

Cause:
`https://nl.alpinelinux.org/alpine/v3.13/main/x86_64/linux-virt-5.10.16-r0.apk` returns `404 Page Not Found`

Ran on:
```
/etc/lsb-release
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=20.1
DISTRIB_CODENAME=ulyssa
DISTRIB_DESCRIPTION="Linux Mint 20.1 Ulyssa"

/etc/upstream-release/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu Focal Fossa"
```

**Comment by hasezoey** [comment]
@mfranciszkiewicz sorry, but building latest commit errors:
```
  wget -q -O "unverified" "https://nl.alpinelinux.org/alpine/v3.13/main/x86_64/linux-virt-5.10.26-r0.apk"
  echo "50b93e671943b9fdec738783ce30ce6379688d0365435d40d94a87b2236e4e30" "unverified" | sha256sum -c || (echo "Kernel apk checksum verification failed!" && exit 1)
  unverified: OK
  rm -f -rf unpacked_kernel
  mkdir unpacked_kernel
  tar --warning=no-unknown-keyword -C unpacked_kernel -vxzf unverified >/dev/null
  rm -f unverified
  cp unpacked_kernel/boot/vmlinuz-virt .
  musl-gcc -std=c11 -O2 -Wall -Wextra -Werror -fPIE -pie -Iinclude/ -DNDEBUG -o src/init.o -c src/init.c
```

**Comment by hasezoey** [comment]
confirmed fixed (also with #84)
